components/common-data-items.js

```diff
@@ -581,6 +581,20 @@ export function FileDataItems({ item, fileSet = null, children }) {
           <DataItemValue>{dataSize(item.file_size)}</DataItemValue>
         </>
       )}
+      {item.anvil_url && (
+        <>
+          <DataItemLabel>AnVIL Url</DataItemLabel>
+          <DataItemValue>
+            <Link href={item.anvil_url}>{item.anvil_url}</Link>
```

Comment:
Use an `<a>` tag with `target="_blank"` and `rel="noopener noreferrer"`. Looks like this has to be an external link, so just use the simpler `<a>` tag instead of the more complex `<Link>` tag that’s for internal links.